tests/test_phase2.py

```
@@ -0,0 +1,292 @@
+
+    # Check if job data is stored in Redis
+    stored_job_data = redis_client.hgetall(f"pytaskflow:job:{job_id}")
+    assert stored_job_data[b'target_function'].decode() == "success_task"
+    assert redis_client.lrange(f"pytaskflow:queue:{job.queue}", 0, -1) == [job_id.encode()]
+
+    dequeued_job = redis_storage.dequeue(["default"], timeout_seconds=1)
+    assert dequeued_job.id == job.id
+    assert dequeued_job.state_name == ProcessingState.NAME # State should change to Processing
+
+    # Check if job is moved to processing list
+    assert redis_client.lrange(f"pytaskflow:queue:processing", 0, -1) == [job_id.encode()]
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)